cleo_plugins/Audio/C3DAudioStream.cpp

```diff
@@ -8,16 +8,19 @@ using namespace CLEO;
 C3DAudioStream::C3DAudioStream(const char* filepath) : CAudioStream()
 {
     // see https://github.com/cleolibrary/CLEO5/pull/230
-    static_assert(offsetof(C3DAudioStream, streamInternal) == 4 && alignof(C3DAudioStream) == 4, "C3DAudioStream compatibility with CLEO4 broken!");
+    static_assert(offsetof(C3DAudioStream, streamInternal) == 4 && alignof(C3DAudioStream) == 4,
+                  "C3DAudioStream compatibility with CLEO4 broken!");
 
     if (isNetworkSource(filepath) && !CSoundSystem::allowNetworkSources)
     {
-        TRACE("Loading of 3d-audiostream '%s' failed. Support of network sources was disabled in SA.Audio.ini", filepath);
+        TRACE("Loading of 3d-audiostream '%s' failed. Support of network sources was disabled in SA.Audio.ini",
+              filepath);
         return;
     }
 
     unsigned flags = BASS_SAMPLE_3D | BASS_SAMPLE_MONO | BASS_SAMPLE_SOFTWARE | BASS_STREAM_PRESCAN;
-    if (CSoundSystem::useFloatAudio) flags |= BASS_SAMPLE_FLOAT;
+    if (CSoundSystem::useFloatAudio)
+        flags |= BASS_SAMPLE_FLOAT;
```

Comment:
I would personally prefer to either 
1. enforce { } on all conditions, e.g.
    if (CSoundSystem::useFloatAudio)
    {
        flags |= BASS_SAMPLE_FLOAT;
    }

2. or never wrap single if condition, e.g. `if (CSoundSystem::useFloatAudio) flags |= BASS_SAMPLE_FLOAT;`
this way it's clear that this line belongs to if.

Reply:
other than that, looks good to me.

Reply:
so the above is a choice between

https://clang.llvm.org/docs/ClangFormatStyleOptions.html#allowshortifstatementsonasingleline
https://clang.llvm.org/docs/ClangFormatStyleOptions.html#insertbraces

InsertBraces has a warning against its usage, so probably allowing short if to stay on the same line is preferred (SIS_WithoutElse)